client/src/test/scala/io/delta/sharing/client/DeltaSharingFileSystemSuite.scala

```diff
@@ -235,7 +235,59 @@ class DeltaSharingFileSystemSuite extends SparkFunSuite {
     }
   }
 
-  test("https traffic is downgraded to http when configured") {
+  test("https traffic is downgraded to http") {
+    val server = new Server(0)
+    val handler = new ServletHandler()
+    server.setHandler(handler)
+    handler.addServletWithMapping(new ServletHolder(new HttpServlet {
+      override def doGet(req: HttpServletRequest, resp: HttpServletResponse): Unit = {
+        resp.setContentType("text/plain")
+        resp.setStatus(HttpServletResponse.SC_OK)
+
+        // scalastyle:off println
```

Comment:
do we still need this in the test?

Reply:
no the scalastyle, but the "hello world" line.

Reply:
Ah. Do you want me to remove this from all the tests?

Reply:
yeah, if it's not used in any test, then we can remove?